lib/acts_as_ar_model.rb

```diff
@@ -1,3 +1,5 @@
+require_relative "extensions/descendant_loader.rb"
```

Comment:
Remove the .rb

Reply:
Following the precedent from my other [recent comment](https://github.com/ManageIQ/manageiq/pull/15460#discussion_r160856836):

* Will remove the `.rb` (for some reason I thought it was required... or helped in some way, but nope)
* Not sure how I will test the autoloading concern to be honest.  Might take me more time to investigate then I probably should at this time, mostly because I only so deep of knowledge on the internal workings of Rails' autoloading that I probably would need to do some spelunking to properly give a "yay" or "nay".  I will try and update regardless.